src/main/kotlin/me/partlysanestudios/partlysaneskies/features/commands/Discord.kt

```diff
@@ -0,0 +1,36 @@
+//
+// Written by Su386.
+// See LICENSE for copyright and license notices.
+//
+
+package me.partlysanestudios.partlysaneskies.features.commands
+
+import me.partlysanestudios.partlysaneskies.PartlySaneSkies
+import me.partlysanestudios.partlysaneskies.PartlySaneSkies.Companion.discordCode
+import me.partlysanestudios.partlysaneskies.PartlySaneSkies.Companion.minecraft
+import me.partlysanestudios.partlysaneskies.commands.PSSCommand
+import net.minecraft.command.ICommandSender
+import net.minecraft.event.ClickEvent
+import net.minecraft.util.ChatComponentText
+import net.minecraft.util.IChatComponent
+
+object Discord {
+    fun registerDiscordCommand() {
+        PSSCommand("discord")
```

Comment:
I wanna argue about this

Reply:
?

Reply:
What is there to argue?

Reply:
naming it "discord"

Reply:
Do you want me to change it?

Reply:
I would rather not have it named discord